### PR TITLE
Builds, docker, docs, and basic app scaffolding

### PR DESCRIPTION
**Pull request by tomthorogood**

### CODE REVIEW

**Review by tomthorogood — COMMENTED** [review]
Just a note that `poetry.lock` is autogenerated and shouldn't be reviewed.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,104 @@
...
+
+Validate everything works by running the pre-push script which:
+
+```
+./scripts/pre-push.sh --no-commit --pro
+```
+
+You should see a message at the end reading:
+ 
+> "🚢 Your commit looks good to go! 🌈"
```
Comment (krlowe) [inline README.md:63]: You should have the :hotdog: emoji in here somewhere, too.

File: docs/docker.md [hunk 2]
```diff
@@ -0,0 +1 @@
+TODO...
```
Comment (krlowe) [inline docs/docker.md:1]: Excellent.  Make a note to check back on this in 2024 to see if it got done.

**Review by krlowe — APPROVED** [review]
Nothing glaring...  Sure, I'll approve it.

File: README.md (hunk 1 above)
Comment (tomthorogood) [reply README.md:63]: 🌭

File: README.md [hunk 3]
```diff
@@ -1,2 +1,104 @@
...
+
+This indicates everything works properly inside of a docker image.
+
+# Key Dependencies
+
+We are using [poetry](https://python-poetry.org/) as a dependency management tool. Poetry takes care of setting up 
+virtual environments and installing dependencies. No more having to muck about with any of that yourself. It is
```
Comment (goulter) [inline README.md:70]: > Poetry takes care of setting up 
virtual environments and installing dependencies. No more having to muck about with any of that yourself.

Curious, I haven't seen that before. Does that mean manually setting up a venv (like we do for`identity-uw`) isn't needed here?

**Review by goulter — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (tomthorogood) [reply README.md:70]: Correct.

**Pull request merged**